frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -6,184 +6,213 @@
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine",
+        "version": "v1"
+      },
+      "component": { "$codeRef": "VirtualizationPage.VirtualizationPage" }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_HAS_v1_API"]
     }
   },
   {
-    "type": "console.catalog/item-type",
+    "type": "console.page/resource/list",
     "properties": {
-      "type": "VmTemplate",
-      "title": "%kubevirt-plugin~Virtual Machines%",
-      "catalogDescription": "%kubevirt-plugin~Quickly create a virtual machine from a template.%",
-      "typeDescription": "%kubevirt-plugin~**Virtual Machines** are templates for quickly creating a virtual machine.%",
-      "filters": [
-        {
-          "label": "%kubevirt-plugin~Template Providers%",
-          "attribute": "templateProvider"
-        }
-      ]
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine",
+        "version": "v1alpha3"
+      },
+      "component": { "$codeRef": "VirtualizationPage.VirtualizationPage" }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_HAS_v1alpha3_API"]
     }
   },
   {
-    "type": "console.catalog/item-provider",
+    "type": "console.navigation/resource-ns",
     "properties": {
-      "catalogId": "dev-catalog",
-      "type": "VmTemplate",
-      "provider": { "$codeRef": "createVM.catalogVMTemplateProvider" }
+      "id": "virtualmachines",
+      "section": "virtualization",
+      "name": "%kubevirt-plugin~VirtualMachines%",
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine",
+        "version": "v1"
+      }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_HAS_v1_API"]
     }
   },
   {
-    "type": "console.catalog/item-filter",
+    "type": "console.navigation/resource-ns",
     "properties": {
-      "catalogId": "dev-catalog",
-      "type": "Template",
-      "filter": { "$codeRef": "createVM.catalogVMTemplateFilter" }
+      "id": "virtualmachines",
+      "section": "virtualization",
+      "name": "%kubevirt-plugin~VirtualMachines%",
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine",
+        "version": "v1alpha3"
+      }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_HAS_v1alpha3_API"]
     }
   },
   {
-    "type": "console.redux-reducer",
+    "type": "console.page/resource/details",
     "properties": {
-      "scope": "kubevirt",
-      "reducer": { "$codeRef": "reduxReducer" }
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine"
+      },
+      "component": { "$codeRef": "VirtualMachinesDetailsPage.VirtualMachinesDetailsPage" }
     },
     "flags": {
       "required": ["KUBEVIRT"]
     }
   },
   {
-    "type": "console.context-provider",
+    "type": "console.navigation/href",
```

Comment:
lets use `console.page/resource` for templates too, 
pros: we will be using the console to create the links and routes
cons: we will need to rethink the list and details pages to not shadow none vm templates